core/src/main/java/com/datastax/oss/driver/internal/core/session/PoolManager.java

```diff
@@ -221,18 +222,23 @@ private void init(CqlIdentifier keyspace) {
       stateEventFilter.start();
 
       Collection<Node> nodes = context.getMetadataManager().getMetadata().getNodes().values();
-      List<CompletionStage<ChannelPool>> poolStages = new ArrayList<>(nodes.size());
+      Map<Node, CompletionStage<ChannelPool>> poolStageByNode = new HashMap<>(nodes.size());
       for (Node node : nodes) {
         NodeDistance distance = node.getDistance();
         if (distance == NodeDistance.IGNORED) {
           LOG.debug("[{}] Skipping {} because it is IGNORED", logPrefix, node);
         } else if (node.getState() == NodeState.FORCED_DOWN) {
           LOG.debug("[{}] Skipping {} because it is FORCED_DOWN", logPrefix, node);
+        } else if (poolStageByNode.containsKey(node)) {
+          LOG.warn("[{}] Skipping {} because pool for it is already created", logPrefix, node);
```

Comment:
Probably sufficient to use debug here :+1:
```suggestion
          LOG.debug("[{}] Skipping {} because pool for it is already created", logPrefix, node);
```

Reply:
Ofc, agree. Changing.